src/main/java/com/github/romankh3/image/comparison/ImageComparison.java

```diff
@@ -173,6 +173,35 @@ public ComparisonResult compareImages() throws IOException {
 
         return ComparisonResult.defaultMisMatchResult(expected, actual).setResult(resultImage);
     }
+    /**
+     * Draw rectangles which cover the regions of the difference pixels.
+     * @param resizeIfSizeMisMatch true if resizing is needed to match expected dimensions.
+     * @return the result of the drawing.
+     * @throws IOException due to saving result image.
+     */
+    public ComparisonResult compareImages(boolean resizeIfSizeMisMatch) throws IOException {
```

Comment:
I'd like you to use main method `compareImages` inside this method after resizing, It will help to avoid code duplicate.

Reply:
I already called the main function after resizing, if that's what you mean.